### ISSUE TITLE
"Localize plugin" function is broken

### ISSUE THREAD

**Issue by 0xb160c0c5** [TES5Edit/TES5Edit#594]
**What is the version of xEdit you are using?**
FO4Edit 3.3.3a BETA (6922BCDD) starting session 2018-11-11 21:14:43

**Describe the bug**
"Localize plugin" function doesn't work.

**To Reproduce**
1.Change "Language for non-Unicode programs" to "Chinese(Simplified, PRC)".
2.Download and open the attachment with FO4Edit.
3.Right click on the mod, choose Other - Localization - Localize plugin.
4.When localization is done, FO4Edit pops up a dialog that says "No mapping for the Unicode character exists in the target multi-byte code page".
5,Save and reopen the mod, the mod remains unlocalized, no "Localized" flag in plugin header.

**Expected behavior**
Mods got properly localized.

**Additional context**
This bug also happens on SSEEdit.
Sometimes if you follow the steps and advance to step 4, the message windows will print "Warning: unused data in XXXX". And when you navigate to the record, FO4Edit would keep poping up a window says "No mapping for the Unicode character exists in the target multi-byte code page" until you kill the process.
In the attachment, "Unlocalized" folder is the original test esps, and the "Localized" folder is the esps processed by FO4Edit on my PC.
[Desktop.zip](https://github.com/TES5Edit/TES5Edit/files/2569403/Desktop.zip)
[FO4Edit_log.txt](https://github.com/TES5Edit/TES5Edit/files/2569404/FO4Edit_log.txt)

**Comment by ElminsterAU** [comment]
1.Change "Language for non-Unicode programs" to "Chinese(Simplified, PRC)".

Could you please stop always starting with that instruction? It has no relevance for 3.3.3 and later. xEdit is not a "non-Unicode program". And with 3.3.3 all ANSI to Unicode conversions are using explicit code pages and never the system ANSI code page.

Please read the detailed instructions the beginning of the What's New document.

Looking at the log you posted:
```
Using language: En
Using general string encoding: 1252  (ANSI - À­¶¡Óï I)
Using translatable string encoding: 1252  (ANSI - À­¶¡Óï I)
Using VMAD string encoding: 65001 (UTF-8)
```
Your  translatable string encoding is set to 1252, because your language is set to En.
I assume the file you actually want to localize is using UTF8.
You can either set your language to something other than En, e.g. `-l:Ch' (or whatever) or you can explicitly set the code page to UTF8 using `-cp-trans:utf8`.

**Comment by 0xb160c0c5** [comment]
Please read from the recent session. In the first session I forgot modify ini.

**Comment by 0xb160c0c5** [comment]
Read this log, it may be clearer.
The log for the second session is not write to the log file as I have to kill the process, see the screenshot.
[FO4Edit_log.txt](https://github.com/TES5Edit/TES5Edit/files/2569503/FO4Edit_log.txt)
![default](https://user-images.githubusercontent.com/26914416/48314671-a740c700-e607-11e8-8ca3-ff25c773d310.jpg)

**Comment by ElminsterAU** [comment]
Already working on fixing the issue...

**Comment by ElminsterAU** [comment]
Please test if https://cdn.discordapp.com/attachments/473350149306056714/511217041143758868/TES5Edit_3.3.3b_BETA.7z solves your problem.

**Comment by 0xb160c0c5** [comment]
xEdit 3.3.3b works well on Fallout 4, but it generates warning and error when localizing SSE esps. See the attachment.
[Desktop.zip](https://github.com/TES5Edit/TES5Edit/files/2569589/Desktop.zip)
[SSEEdit_log.txt](https://github.com/TES5Edit/TES5Edit/files/2569591/SSEEdit_log.txt)
[SSEEditException.log](https://github.com/TES5Edit/TES5Edit/files/2569592/SSEEditException.log)

**Comment by ElminsterAU** [comment]
Then the strings in the source file do not contain valid UTF8 strings.

**Comment by 0xb160c0c5** [comment]
I've tested again and the bug never happens again.
After all, the bug seems to be solved.

**Comment by ElminsterAU** [comment]
The commit referenced above should solve this problem. It's planned to be included in 3.3.4.
